frontend/src/pages/DeviceSettingsPage/DeviceSettingsPage.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { ApplicationState, Dispatch } from '../../store'
+import { DeviceHeaderMenu } from '../../components/DeviceHeaderMenu'
+import { List, Typography } from '@material-ui/core'
+import { DeviceNameSetting } from '../../components/DeviceNameSetting'
+import { TestUI } from '../../components/TestUI'
+import { Gutters } from '../../components/Gutters'
+import { TagEditor } from '../../components/TagEditor'
+import { ListItemSetting } from '../../components/ListItemSetting'
+
+type Props = {
+  targetDevice: ITargetDevice
+  device: IDevice
+}
+
+enum notificationType {
+  'ONW' = 'onlineDeviceNotification',
+  'DEVICE_SHARED' = 'onlineSharedDeviceNotification',
+  'EMAIL' = 'notificationEmail',
+}
+
+const NOTIFICATION = {}
+export const DeviceSettingsPage: React.FC<Props> = ({ targetDevice, device }) => {
```

Comment:
Lets change this to a `<NotificationSettings device={device} />` component and import into the `DeviceEditPage.tsx` at the bottom. You can then delete the DeviceSettingsPage.